source/index.erb

```diff
@@ -25,6 +25,7 @@ title: Energy Globe Award
         <li ng-repeat="year in filters.year.names">
           <input type="checkbox" id="year_checkbox_{{year}}" ng-model="filters.year.filterMap[year]">
           <label for="year_checkbox_{{year}}">{{year}}</label>
+          <%= sprockets.find_asset('checkbox.svg').to_s %>
```

Comment:
Nice approach! What makes my heart cry a little, though, is that always the same SVG needs to be repeatedly put into the DOM. The only alternative I could think of would be to insert the SVG with JavaScript. But this would be overkill for now.